msal-auth-node/app.js

```diff
@@ -0,0 +1,74 @@
+/*
+ This application demonstrates how to include authorization
+ to an API written in Node.Js 16.
+*/
+
+// Microsoft Authentication Library (MSAL) for Node.Js
+const msal = require('@azure/msal-node')
+
+// Node.Js Express Framework
+const express = require('express')
+
+// The port the express server will listen on
+const SERVER_PORT = 8080
+
+const config = {
+  auth: {
+    
+    // 'Application (client) ID' of app registration in Azure portal - this value is a GUID
+    clientId: '',
+    
+    // Client secret 'Value' (not the ID) from 'Client secrets' in app registration in Azure portal
+    clientSecret: '',
+
+    // Full directory URL, in the form of https://login.microsoftonline.com/<tenant>
+    authority: ``
```

Comment:
I'm guessing this was like this in the other PR, and I know it was because this used to have a concat in it.  But we can probably set it back to this in here and in the README

```suggestion
    authority: ''
```